Microsoft.Recognizers.Text.Number/BaseNumberRecognizer.cs

```diff
@@ -1,27 +1,28 @@
-﻿using System;
-using System.Collections.Generic;
-
-namespace Microsoft.Recognizers.Text.Number
-{
-    public abstract class BaseNumberRecognizer : Recognizer<Dictionary<string, Dictionary<Type, IModel>>>
-    {
-        public static IModel GetModel<TModel>(string culture, bool fallbackToDefaultCulture = true)
-        {
-            if (!ModelInstances.ContainsKey(culture) || !ModelInstances[culture].ContainsKey(typeof(TModel)))
-            {
-                if (fallbackToDefaultCulture)
-                {
-                    culture = DefaultCulture;
-                }
-                else
-                {
-                    throw new Exception($"ERROR: No IModel instance for {typeof(TModel)}");
-                }
-            }
-
-            var model = ModelInstances[culture][typeof(TModel)];
-
-            return model;
-        }
-    }
-}
+using System;
```

Comment:
In Microsoft/Recognizers-Text, there is a text project. Most of your change is deleting the old code and adding new code which is the same as the old code. I think you can abandon this pr.